### ISSUE TITLE
[BUG] M871 command miss first index value

### ISSUE THREAD

**Issue by BastR** [MarlinFirmware/Marlin#18908]
### Bug Description

I try to enter manually (with the 871 command) the compensation value for the pinda probe but I can't specify the first value of the table.  

Index 1 (I1) is always the second value of the table, I tried I0 just in case, but it's not possible.

<img width="983" alt="Capture d’écran 2020-08-03 à 10 04 13" src="https://user-images.githubusercontent.com/20129420/89160571-91a8c900-d571-11ea-8a6a-bb62ab9e9510.png">

@tompe-proj

**Comment by boelle** [comment]
@BastR 

Configurations, please
Please ZIP up your `Configuration.h` and `Configuration_adv.h` files (as
requested in the Issue template) and drop them into your next reply.
We'll check them over and see if anything is amiss.

**Comment by BastR** [comment]
My config is here: https://github.com/BastR/Marlin/tree/V-Core-Pro/Marlin

**Comment by TheNeskik** [comment]
If I'm not wrong, the temp probe compensation need to have your Z-offset well calibrate before doing G76.
When you do your Z-offset calibration, you need to do it with your bed set to the first temperature of your bed compensation and your probe set to the first value of the table. (Possibly, we can admit that below 50°C for the bed and 30°C for the probe their is no shift). That's why your index 0 (I0) can't be edit because you have already calibrate your printer for this temp.

that's what i always thought

**Comment by BastR** [comment]
That makes sense, but I'd like confirmation. 

My Z-offset is well calibrated, however I can't get the automatic calibration (G76) to work, I tried it 3 times and each time the process goes on hold and I have to reset the machine. It's quite hot at home at the moment, I don't know if it's because of that, that's why I wanted to enter the compensation values myself using the table of an original pinda V2 provided here: https://marlinfw.org/docs/features/probe_temp_compensation.html

**Comment by rjfsilva** [comment]
I have not tried to enter a compesation value manually yet, but I can confirm the same behaviour when I try the automatic calibration (G76).

**Comment by BastR** [comment]
I have an original pinda V2 and it's impossible for me to make a self-calibration, the temperature of the pinda never goes back under 30°C. The only way I found is to put it in contact with a cold object coming out of the fridge :p and wait.

I've tried to change #define PTC_SAMPLE_START 30.0f du 35.0f but it doesn't work.

Changing PTC_SAMPLE_START 30.0f to 35.0f change the first value of the compensation table to 35 instead of 30°C, that's why I couldn't define a compensation value for 30°C.

Now I set PTC_SAMPLE_START back to 30°C and manually entered the values in the table without any issues, I will try an automatic compensation this winter.
